### PR TITLE
max solution

### PR DESCRIPTION
**Pull request by ArnyCorp**
https://arnycorporation.github.io/react_static-list-of-todos/

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,3 +1,9 @@
 .App {
   text-align: center;
 }
+
+.wrapper {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+  grid-gap: 20px;
+}
```
Comment (irenhh) [inline src/App.css]: new line

File: src/App.jsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+function getTodoWithUser(todosArr, usersArr) {
+  return todosArr.map(todo => ({
+    ...todo,
+    user: usersArr.find(user => user.id === todo.userId),
+  }));
+}
+
+function App() {
+  const preparedTodos = getTodoWithUser(todos, users);
```
Comment (irenhh) [inline src/App.jsx]: move this outside the function

File: src/components/User/User.jsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import './User.scss';
+import { UserProps } from '../../constants/PropTypes';
+
+function User({
+  name, username, email, completed,
+}) {
+  return (
+
```
Comment (irenhh) [inline src/components/User/User.jsx]: remove unnecessary line

File: src/components/UsersList/UsersList.jsx [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+import users from '../../api/users';
+
+function Users() {
+  return (
+    <p>
+      <span>Users: </span>
+      <ul className="users">
+        {users.map(user => (
+          <li>
+            {' '}
```
Comment (irenhh) [inline src/components/UsersList/UsersList.jsx]: use template string

**Comment by ArnyCorp** [comment]
> https://prnt.sc/p44zga remove this space after `X`

**Pull request closed without merging**